cmake/Modules/drishti_set_unit_test_assets.cmake

```diff
@@ -1,52 +0,0 @@
-function(drishti_set_unit_test_assets
-    DRISHTI_ACF_FACE_MODEL
-    DRISHTI_FACE_LANDMARKER
-    DRISHTI_EYE_MODEL
-    DRISHTI_MEAN_FACE_LANDMARKS
-    )
-
-  set(ext "")
-  if(DRISHTI_SERIALIZE_WITH_CEREAL)
```

Comment:
The `DRISHTI_SERIALIZE_WITH_CVMATIO` is currently used to deserialize models generated in Matlab code (the `ACF` training is still performed in Matlab and hasn't been implemented in C++).  We'll need to keep this around, and it is fairly easy to turn on and off as need.  I think it makes sense to remove boost serialization at some point in the near future (`DRISHTI_SERIALIZE_WITH_BOOST` and the `DRISHTI_USE_TEXT_ARCHIVES` (added for MSVC)), such that `DRISHTI_SERIALIZE_WITH_CEREAL` will be the default model (non-optional).  This will simplify a fair amount of code (boost stuff is mostly supporting legacy models at this point).  I'd like to review this in more detail before blowing away the code.  We can deprecate and remove any boost specific unit tests for now, but let's keep the serialization and related templates around for a little while longer.  I'll create an issue specific to removing boost serialization and summarize TODO items there.